boto3/docs/method.py

```diff
@@ -59,3 +59,14 @@ def document_model_driven_resource_method(
         new_return_section.write(
             ':returns: %s' % return_description)
         new_return_section.style.new_line()
+
+
+def _method_returns_collection(resource):
```

Comment:
I'm fine with keeping it in ``method.py`` because it is not needed in any other modules.

The name is a little misleading here. I would not call it as returning a collection. That is a whole other resource concept. This is more just checking if a list of resources is returned. Maybe call it ``_method_returns_resource_list()``?